### DISCUSSION TITLE
How to open localhost

### DISCUSSION THREAD

**Discussion by suzhiking** [chase/awrit#45] [Q&A]
Hi, I tried to open locahost in awrit but it only gives a blank page, but it works fine with other links. How can I fix it?

<img width="1728" alt="Screenshot 2024-10-08 at 7 44 41 PM" src="https://github.com/user-attachments/assets/f0f61c16-0d8a-48d9-a399-9d943ecc0ddc">

**Comment by chase** [comment]
What URL are you using for localhost exactly? You'll need to use `http://localhost` in the latest version, but please give it a try again with the latest version.
